setup.py

```diff
@@ -82,3 +81,17 @@
     # zip_safe=False,
     # todo: add classifiers
 )
+
+# Naming format of the pypi wheel
+wheel_name_format = "{name}-{version}-{py_version}-none-any.whl"
```

Comment:
@FanwangM currently I am using naming conventions that will generate packages like this `qc_selector-0.0.1-py3-none-any.whl`. I used [streamlit's](https://pypi.org/project/streamlit/#files) naming conventions as a reference. If you want some other format, let me know, I will update it!